forge-gui/res/cardsfolder/s/silvanuss_invoker.txt

```diff
@@ -0,0 +1,7 @@
+Name:Silvanus's Invoker
+ManaCost:2 G
+Types:Creature Dragon Druid
+PT:3/2
+A:AB$ Untap | Cost$ 8 | ValidTgts$ Land | TgtPrompt$ Select target land | SubAbility$ DBAnimate | PrecostDesc$ Conjure Elemental - | SpellDescription$ Untap target land you control. It becomes an 8/8 Elemental creature with trample and haste until end of turn. It's still a land.
+SVar:DBAnimate:DB$ Animate | Defined$ Targeted | Power$ 8 | Toughness$ 8 | Keywords$ Trample & Haste | Types$ Creature,Elemental |
+Oracle:Conjure Elemental - {8}: Untap target land you control. It becomes an 8/8 Elemental creature with trample and haste until end of turn. It's still a land. 
```

Comment:
Add
`DeckHas:Type$Elemental`
so the AI will consider add this to Elemental synergy decks